### PR TITLE
Add ergosaurus default keymap

### PR DESCRIPTION
**Pull request by cfbender**
Following https://github.com/qmk/qmk_firmware/pull/8013, I figured I would go ahead and add this here. Hope the formatting and everything is correct!

### CODE REVIEW

File: public/keymaps/ergosaurus_default.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "keyboard": "ergosaurus",
+    "keymap": "default_6d5812a",
+    "commit": "6d5812a86c7eaa3e84d8a69126aa04195713be62",
+    "layout": "DEDAULT",
```
Comment (noroadsleft) [inline public/keymaps/ergosaurus_default.json]:
```suggestion
    "layout": "LAYOUT",
```

File: public/keymaps/ergosaurus_default.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        "KC_TRNS",    "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_LEFT", "KC_RGHT",            "KC_TRNS",
+                      "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_DOWN", "KC_TRNS", "KC_TRNS",
+                      "KC_TRNS",            "KC_TRNS",            "KC_TRNS", "KC_TRNS",            "KC_TRNS",            "KC_TRNS",                                  "KC_TRNS"
+      ],
+      [
+        "KC_TRNS",    "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",  "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+        "KC_TRNS",    "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_UP",   "KC_TRNS",  "KC_TRNS", "KC_TRNS",   "KC_TRNS", "KC_TRNS",
+        "KC_TRNS",    "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_LEFT", "KC_DOWN", "KC_RIGHT", "KC_LEFT", "KC_TRNS",           "KC_TRNS",
```
Comment (noroadsleft) [inline public/keymaps/ergosaurus_default.json]:
```suggestion
        "KC_TRNS",    "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_LEFT", "KC_DOWN", "KC_RGHT", "KC_TRNS", "KC_TRNS",           "KC_TRNS",
```

**Comment by cfbender** [comment]
Thanks @noroadsleft, and thanks for your fix in the main repo as well!

**Comment by noroadsleft** [comment]
You're welcome, and thanks for this!

**Pull request merged**